scripts/babel/rewrite-modules.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright 2013-2015, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+'use strict';
+
+function mapModule(context, module) {
+  // Look in options passed to babel for `_moduleMap` which contains a map of
+  // replacements. This allows other packages to be published to npm and used
+  // directly without being a part of the same build.
+  var moduleMap = context.state.opts._moduleMap || {};
+  if (moduleMap.hasOwnProperty(module)) {
+    return moduleMap[module];
+  }
+  // Jest understands the haste module system, so leave modules intact.
+  if (process.env.NODE_ENV !== 'test') {
```

Comment:
Is there by any chance a better way to detect a jest environment? This looks like it might lock project that uses fbjs into jest for testing (since `require` calls wont be transformed in any test environment).